ui/pages/routes/routes.component.js

```diff
@@ -752,6 +796,25 @@ export default class Routes extends Component {
               onClose={() => this.setState({ hideConnectAccountToast: true })}
             />
           ) : null}
+          {showAutoNetworkSwitchToast ? (
+            <Toast
+              key="switched-network-toast"
+              startAdornment={
+                <AvatarNetwork
+                  size={AvatarAccountSize.Md}
+                  borderColor={BorderColor.transparent}
+                  src={switchedNetworkDetails?.imageUrl}
+                />
+              }
+              text={this.context.t('switchedNetworkToastMessage', [
+                switchedNetworkDetails.nickname.toUpperCase(),
+                getURLHost(switchedNetworkDetails.origin).toUpperCase(),
```

Comment:
Nit: Interesting choice for this to be in all-caps, what was the reasoning there? I never see URLs/origins depicted as upper-case, it's always lower-case (e.g. in the address bar, they're normalized to be lower-cased).

Reply:
Design decision, but I'll follow up:

<img width="328" alt="SCR-20240327-jxyd" src="https://github.com/MetaMask/metamask-extension/assets/46655/f4f53c12-3a1e-4d05-8097-97c4524953b4">

Reply:
@mmragkandala says this can be lowercase, so I've removed the uppercase'ing